ownCloud/Tools/URL+Extensions.swift

```diff
@@ -44,4 +44,53 @@ extension URL {
 
 		return progress
 	}
+    
+    func privateLinkItemID() -> String? {
+        
+        if self.pathComponents.count > 2 {
+            if self.pathComponents[self.pathComponents.count - 2] == "f" {
```

Comment:
Please can you add a comment, why you searching for "f" or give an example.

Reply:
@hosy done in last commit